source/scripts/components/Snackbar/snackbar.css

```diff
@@ -0,0 +1,42 @@
+.snackbar {
+  position: fixed;
+  top: 90vh;
```

Comment:
I think `bottom: 10vh` suits here better. Imagine you have a lot of content to show in your Snackbar. With `top: 90vh` it will grow downward and it'll have only `10vh` of free space. With `bottom: 10vh` it'll grow upward and it'll have `90vh` of free space to show content.